xml/x509_jarm.xml

```diff
@@ -0,0 +1,139 @@
+    <example>2ad2ad0002ad2ad0002ad2ad2ad2ada9e96d3ba9f7903758a731e0fa01453d</example>
+    <param pos="0" name="os.device" value="NAS"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="DSM"/>
+    <param pos="0" name="os.vendor" value="Synology"/>
+    <param pos="0" name="hw.vendor" value="Synology"/>
+    <param pos="0" name="hw.device" value="NAS"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb$">
+    <description>Proxmox open-source virtualization platform</description>
+    <example>2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb</example>
+    <param pos="0" name="service.vendor" value="proxmox"/>
+    <param pos="0" name="service.product" value="proxmox"/>
+    <param pos="0" name="os.vendor" value="Proxmox"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Proxmox"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef$">
+    <description>Ubiquiti EdgeRouter</description>
+    <example>2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef</example>
+    <param pos="0" name="hw.vendor" value="Ubiquiti"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="0" name="hw.Product" value="EdgeRouter X"/>
+    <param pos="0" name="os.vendor" value="Ubiquiti"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.device" value="Router"/>
+  </fingerprint>
+
+  <fingerprint pattern="^21d14d00021d21d21c21d14d21d21d">
+    <description>UniFi Video Controller</description>
+    <example>21d14d00021d21d21c21d14d21d21d16e53b2674d38f90c8e876dc8f3c879c</example>
+    <example>21d14d00021d21d21c21d14d21d21d3fe87b802002478c27f1c0da514dbf80</example>
+    <param pos="0" name="hw.vendor" value="Ubiquiti"/>
+    <param pos="0" name="hw.device" value="Video controler"/>
+    <param pos="0" name="hw.product" value="UniFi Video Controller"/>
+    <param pos="0" name="hw.certainty" value="0.50"/>
+    <param pos="0" name="os.vendor" value="Ubiquiti"/>
+    <param pos="0" name="os.certainty" value="0.50"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="UniFi Video Controller"/>
+  </fingerprint>
+
+  <fingerprint pattern="^07d14d16d21d21d00042d43d000000aa99ce74e2c6d013c745aa52b5cc042d$">
+    <description>Metasploit listener</description>
+    <example>07d14d16d21d21d00042d43d000000aa99ce74e2c6d013c745aa52b5cc042d</example>
+    <param pos="0" name="service.vendor" value="Rapid7"/>
+    <param pos="0" name="service.product" value="Metasploit"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:rapid7:metasploit:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^07d14d16d21d21d07c42d41d00041d24a458a375eef0c576d23a7bab9a9fb1$">
+    <description>Cobalt Strike listener</description>
```

Comment:
We're probably going to get false positives here.
https://blog.cobaltstrike.com/2020/12/08/a-red-teamer-plays-with-jarm/